src/init/welcome.ts

```diff
@@ -14,11 +14,8 @@ export function manageWelcomeMessage(settingsManager: ISettingsManager) {
     return;
   }
 
-  if (settingsManager.isNewVersion()) {
-    settingsManager.updateStatus(state.status);
-    if (!getConfig().vsicons.dontShowNewVersionMessage) {
-      showNewVersionMessage(settingsManager);
-    }
+  if (!getConfig().vsicons.dontShowNewVersionMessage) {
```

Comment:
This message should only be shown when a new version is delivered. Now it will be shown always unless the `dontShowNewVersionMessage` preset is set to false.